core/src/main/scala/org/apache/spark/util/SerializableBuffer.scala

```diff
@@ -20,13 +20,16 @@ package org.apache.spark.util
 import java.io.{EOFException, IOException, ObjectInputStream, ObjectOutputStream}
 import java.nio.ByteBuffer
 import java.nio.channels.Channels
+import com.esotericsoftware.kryo.{Kryo, KryoSerializable}
```

Comment:
Super-minor nit, but there should be an extra newline before the `com.*` import group here.